quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliPomIntegrityIT.java

```diff
@@ -0,0 +1,53 @@
+package io.quarkus.ts.quarkus.cli;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import jakarta.inject.Inject;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.QuarkusCliClient;
+import io.quarkus.test.bootstrap.QuarkusCliRestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+
+@QuarkusScenario
+public class QuarkusCliPomIntegrityIT {
+    /*
+     * This scenario is related to the backport https://github.com/quarkusio/quarkus/issues/39088
+     *
+     */
+    private static final String NEW_EXTENSION = "quarkus-kafka-client";
+    private static final String COMMENT = "<!-- Disable native build on this module -->";
+
+    private static final String APP_NAME = "my-quarkus-app";
+
+    @Inject
+    static QuarkusCliClient cliClient;
+
+    @Disabled
```

Comment:
This should be enabled as https://github.com/quarkusio/quarkus/pull/40869 was merged in main last week + is marked for backport into 3.8 branch (will be part of 3.8.5 release)